packages/salesforcedx-utils-vscode/src/helpers/validations.ts

```diff
@@ -27,3 +27,7 @@ export function isIntegerInRange(
 export function isAlphaNumString(value: string | undefined): boolean {
   return value !== undefined && !/\W/.test(value);
 }
+
+export function isAlphaNumSpaceString(value: string | undefined): boolean {
+  return value !== undefined && /^\w+( *\w*)*$/.test(value);
```

Comment:
This regex allows for multiple spaces and words. 

Note this is not enforcing a user to have a alphanumeric character at the end of their alias. If we change this regex to be '/^\w+( +\w+)*$/`, it will mean the user is displayed with an error while they are in the middle of typing an alias that has a space (Example "Alias ").

Reply:
Does the CLI support having a space at the end? I'm having trouble setting an alias with one

Reply:
Yes they do.

08:38:37.473 sfdx force:org:create -f config/project-scratch-def.json --setalias example with spaces  --durationdays 1 --setdefaultusername --json --loglevel fatal ended with exit code 0

I'm able to open this org and switch to it from the org picker.

Reply:
Cool. Also, not an issue but pointing out that having a space at the beginning does invalidate the rest of the alias name.

Reply:
Good to know. This regex will require users to start off with an alphanumeric character, so we should be good there.

Reply:
I really want to meet that person who wants to name something starting with a space :)